docs/developer-guide/syntax-reference.rst

```diff
@@ -1362,6 +1362,84 @@ Scalar functions
+|                        |                                                                           | in the path portion of a URL.                     |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | Input: ``url encoded``                            |
+|                        |                                                                           | Output: ``'url%20encoded``                        |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| URL_EXTRACT_FRAGMENT   |  ``URL_EXTRACT_FRAGMENT(url)``                                            | Extract the fragment portion of the specified     |
+|                        |                                                                           | value. Returns NULL if ``url`` is not a valid URL |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | Input: ``http://test.com?query#frag``,            |
+|                        |                                                                           | Output: ``frag``                                  |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| URL_EXTRACT_HOST       |  ``URL_EXTRACT_HOST(url)``                                                | Extract the host-name portion of the specified    |
+|                        |                                                                           | value. Returns NULL if the ``url`` is not a valid |
+|                        |                                                                           | URI according to RFC-2396.                        |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | Input: ``http://test.com?query#frag``,            |
+|                        |                                                                           | Output: ``test.com``                              |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| URL_EXTRACT_PARAMETER  |  ``URL_EXTRACT_PARAMETER(url, parameter_name)``                           | Extract the value of the requested parameter from |
```

Comment:
Does any encoding of the param name happen? Or decoding of the result? If so document and update the example to demos this.